tests/aliases.test.sh

```
@@ -0,0 +1,180 @@
+echo
+echo "== A4: uninstall-aliases is marker-scoped =="
+# Without marker scoping, uninstall could delete user-authored files that
+# happen to share a short name. Assert that uninstall reads/checks the
+# marker before removing.
+assert_grep "$UNINSTALL_CMD" \
+  'grep|-q|managed-by' \
+  "uninstall-aliases checks for the marker before removing files"
+
+# Belt-and-braces: uninstall must NOT do an unguarded rm of all 5 paths.
+assert_grep_not "$UNINSTALL_CMD" \
+  'rm[[:space:]]+-f[[:space:]]+"?\$HOME/\.claude/commands/(issue|solve|turbo|simplify|review-pr)\.md"?[[:space:]]*$' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)